pandas/_libs/src/ujson/python/date_conversions.c

```diff
@@ -54,8 +54,8 @@ char *int64ToIso(int64_t value, NPY_DATETIMEUNIT base, size_t *len) {
         PyErr_NoMemory();
         return NULL;
     }
-
-    ret_code = make_iso_8601_datetime(&dts, result, *len, base);
+    ret_code = make_iso_8601_datetime(&dts, result, *len,
```

Comment:
can you make the comment inline with `//` rather than this as its very hard to read

Reply:
@lithomas1 you *might* be able to just run clang-format to do this automatically while cleaning up the rest of the code. Can't remember if done to this file already but definitely done with other files in the past

[28144](https://github.com/pandas-dev/pandas/pull/28144#issuecomment-524687696)

Reply:
I guess cpplint didn't catch this one 🤷 . I develop mainly on Windows so I don't really use clang(Can one install clang-format without clang?).

Anyways, fixed the formatting.